### ISSUE TITLE
Renaming a module doesn't rename open references in other project

### ISSUE THREAD

**Issue by CraigStuntz** [fsprojects/VisualFSharpPowerTools#968]
Project one:

```fsharp
module Foo

let bar a = a
```

Project two:

```fsharp
module FooTests

open Foo
open NUnit.Framework
open FsUnit

[<Test>]
let ``identity`` () = 
    Foo.bar 3 |> should equal 3
```
Build, place cursor on Foo module name in Project 1, rename to ```NewName```.

**Expected** Both ```Foo```s in ```FooTests``` changed to ```NewName```.
**Actual** The line in the test method gets its reference renamed, but the ```open``` line does not get its reference renamed.

**Comment by dungpa** [comment]
Thanks for the bug report.

The quick answer is that there is no symbol for open statements coming from the compiler; therefore, there is no info for us to rename `open Foo` in a reliable way. See relevant feature request at https://github.com/fsharp/FSharp.Compiler.Service/issues/91.

**Comment by exercitusvir** [comment]
I am having this problem even within a project. The files of the renamed module and the not-renamed module are in a folder - the same folder - in case it is related to folders.

**Comment by dungpa** [comment]
@exercitusvir No, it isn't related to folders. As I said above, it's a limitation of FSharp.Compiler.Service.
